docs/client_install/src/asciidoc/_chapters/preparation.adoc

```diff
@@ -220,7 +219,7 @@ $ unzip trafci.zip -d $HOME/trafodion/trafci
 .
 $ cd ..
 $ ls
-apache-trafodion_clients-2.0.1-incubating.tar.gz  clients  jdbct4  trafci
+apache-trafodion_clients-2.2.0.tar.gz  clients  jdbct4  trafci

```

Comment:
Release 2.2 hasn't been officially approved yet... should this say, 2.1.x?

Reply:
This is example in documentation on master branch, for next release. Not for past releases that were incubating releases.